R/mlr_graphs.R

```diff
@@ -0,0 +1,56 @@
+#' @title Dictionary of (sub-)graphs
+#'
+#' @usage NULL
+#' @format [`R6Class`] object inheriting from [`mlr3misc::Dictionary`].
+#'
+#' @description
+#' A simple [`Dictionary`][mlr3misc::Dictionary] storing objects of class [`Graph`].
+#' The dictionary contains a collection of often-used graph structures, and it's aim
+#' is solely to make often-used functions more accessible.
+#' Each `Graph` has an associated help page, which can be accessed via `?<key>`, i.e.
+#' `?bagging_pipeline`.
+#'
+#' @section Methods:
+#' Methods inherited from [`Dictionary`][mlr3misc::Dictionary], as well as:
+#' * `add(key, value)`\cr
+#'   (`character(1)`, `R6ClassGenerator`, `NULL` | `list`)\cr
+#'   Adds constructor `value` to the dictionary with key `key`, potentially
+#'   overwriting a previously stored item.
+#'
+#' @section S3 methods:
+#' * `as.data.table(dict)`\cr
+#'   [`Dictionary`][mlr3misc::Dictionary] -> [`data.table::data.table`]\cr
+#'   Returns a `data.table` with column `key` (`character`).
+#' @family mlr3pipelines backend related
+#' @family Graph
```

Comment:
does this family have another member? If not I'd probably remove this. I don't think it should be put together with the `?Graph` help page.
Maybe instead make a `@family Dictionaries` and add this and the `mlr_pipeops` dict.